### PR TITLE
Mapping showers, washing machines, linen bins

### PR DESCRIPTION
**Pull request by CyberTropic**
## About the PR

Adds showers, washing machines, linen bins across maps.
Adds some missing toilets and a couple missing things spotted near the washrooms.
Fixes door rotations on LV.

## Why / Balance

Parity.

## Media

cap1
cap2
cap4
cap5
image

## Requirements

- [x] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Added washing machines, showers, linen bins, and soaps to the maps
- fix: Fixed incorrect door rotations on LV

### CODE REVIEW

File: Resources/Maps/_RMC14/prison.yml [hunk 1]
```diff
@@ -90321,6 +90858,16 @@ entities:
...
+  - uid: 19554
+    components:
+    - type: Transform
+      pos: -11.519007,58.522633
+      parent: 1
+  - uid: 19555
+    components:
+    - type: Transform
+      pos: -18.52556,60.54068
+      parent: 1
```
Comment (CyberTropic) [inline Resources/Maps/_RMC14/prison.yml:90870]: These are missing tripods

File: Resources/Maps/_RMC14/trijent.yml [hunk 2]
```diff
@@ -100410,6 +100609,18 @@ entities:
...
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 174.5,-151.5
+      parent: 1
+  - uid: 30425
+    components:
+    - type: Transform
+      rot: 1.5707963267948966 rad
+      pos: 173.5,-151.5
+      parent: 1
```
Comment (CyberTropic) [inline Resources/Maps/_RMC14/trijent.yml:100623]: These are reinforced windows

**Comment by CyberTropic** [comment]
Merge conflicts resolved 👍

**Comment by CyberTropic** [comment]
UID collisions resolved by pushing every colliding ID up by 1

**Comment by DrSmugleaf** [comment]
I'll do this in my next mapping PR so you don't have to go through conflict hell, I'll fix the doors too ty

**Pull request closed without merging**